djangocms_link/fields.py

```diff
@@ -1,9 +1,15 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
-
 from django.conf import settings
 
-if 'django_select2' in settings.INSTALLED_APPS:
+
+ENABLE_SELECT2 = getattr(
+    settings,
+    'DJANGOCMS_LINK_SELECT2',
+    False
+)
+
+
+if 'django_select2' in settings.INSTALLED_APPS and ENABLE_SELECT2:
```

Comment:
pro-tip: Maybe not very relevant here but always check the cheapest condition first in order to fail fast and avoid checking the other most expensive condition. So checking `ENABLE_SELECT2` is cheaper than `'django_select2' in settings.INSTALLED_APPS`. As mentioned, is not very relevant here but just advice.